parser/parser.py

```diff
@@ -6,6 +6,7 @@
 import subprocess
 from bs4 import BeautifulSoup
 import xml.dom.minidom
+import re
```

Comment:
@AbegaM - I don't think this import is used. I think we can delete it.

Reply:
Ahh, i forgot to remove that, i will do it now